explainer.html

```diff
@@ -210,56 +201,23 @@ <h2>Defining an RDF Dataset Hash and/or Signing RDF Datasets</h2>
         serialize <span class='rdf'>C(R)</span> to quads&nbsp;[[n-quads]] and sort the resulting set of quads;
       </li>
       <li id=hash>
-        apply a (traditional) hashing function <span class='rdf'>h</span> on the result of the serialization to yield <span class='rdf'>h(R)</span> (the cryptographic hash of the Dataset).
-      </li>
-      <li id=sign>
-        apply a digital signature function on <span class='rdf'>h(R)</span>.
+        apply a (traditional) hashing function <span class='rdf'>h</span> on the result of the serialization to yield <span class='rdf'>h(R)</span>&thinsp;; this can be considered as the cryptographic hash of the Dataset.
       </li>
-    </ol>
+     </ol>
+     <p>
+       The second step, i.e., the sorting of the serialized dataset in quads, also requires the specification of what could be considered as a “canonical” version of N-Quads [[n-quads]] files (handling of white spaces, specifying the exact sorting algorithms to be used, canonical representation of datatype literals, etc.). Considering the simplicity of the N-Quads format, this does not necessitate a significant specification effort, but has a value in its own right. That being said, there may be other approaches to define a hash that do not necessarily involve a sorted N-Quads representation: the Working Group will have to determine the best approach.
```

Comment:
This looks fine as a replacement for the snippet that was being wordsmithed in the now deleted section below...